packages/webapp/pages/feeds/new.tsx

```diff
@@ -130,6 +137,20 @@ const NewFeedPage = (): ReactElement => {
     completeAction(ActionType.CustomFeed);
   }, [completeAction]);
 
+  const trackedStartEventRef = useRef(false);
+
+  useEffect(() => {
+    if (trackedStartEventRef.current) {
+      return;
+    }
+
+    trackedStartEventRef.current = true;
+
+    trackEvent({
+      event_name: AnalyticsEvent.StartCustomFeed,
+    });
+  }, [trackEvent]);
```

Comment:
Calling it in `/feeds/new` just so we don't have to call this even all over where you can press "New feed" on mobile/desktop etc.